main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -70,6 +70,8 @@ object Simplifier extends Phase[TypedAst.Root, SimplifiedAst.Root] {
 
       case TypedAst.Expression.Def(sym, tpe, loc) => SimplifiedAst.Expression.Def(sym, tpe, loc)
 
+      case TypedAst.Expression.Sig(sym, tpe, loc) => ??? // MATT later
```

Comment:
Here would be a good place to replace Sig by HoleError or something like that. Or alternatively throw an internal compiler exception.